Cargo.toml

```diff
@@ -56,6 +57,7 @@ std = [
 alloc = ["dusk-bls12_381/alloc", "msgpacker", "miniz_oxide", "sha2"]
 debug = ["dusk-cdf", "backtrace"]
 rkyv-impl = ["dusk-bls12_381/rkyv-impl", "dusk-jubjub/rkyv-impl", "rkyv", "bytecheck"]
+zeroize = ["dep:zeroize"]
```

Comment:
Since I added zeroize as an optional dependency, I thought that creating a feature like this will make sure that the zeroize crate won't be compiled unless the zeroize feature is enabled? Am I understanding this incorrectly?

Reply:
Oh wait, I can use optional dependencies just like features, right?